src/views/core/members/Single.vue

```diff
@@ -38,6 +38,13 @@
             </button>
           </div>
 
+          <div class="field is-grouped" v-if="can.edit">
```

Comment:
For what we discussed today; if you add to this that you can only see it when gsuite_id is not empty then we can merge this soon